recipes/libcurl/all/conanfile.py

```diff
@@ -192,11 +192,11 @@ def _get_configure_command_args(self):
         if tools.cross_building(self.settings):
             if self.settings.os == "Linux" and "arm" in self.settings.arch:
                 params.append('--host=%s' % self._get_linux_arm_host())
-            elif self.settings.os == "iOS": 
```

Comment:
this also might be an easy hook check (trailing spaces)

Reply:
Yes, that's a good hook, or, check if pylint can verify it.